pickhardtpayments/ChannelGraph.py

```diff
@@ -46,3 +53,66 @@ def get_channel(self, src: str, dest: str, short_channel_id: str):
         if self.network.has_edge(src, dest):
             if short_channel_id in self.network[src][dest]:
                 return self.network[src][dest][short_channel_id]["channel"]
+
+
+def lnd2cln_json(channel_graph_json):
+    """
+    Converts the channel graph json from the LND format to the c-lightning format
+    """
+
+    LND_CLN_POLICY_MAP = {
+        "time_lock_delta": "delay",
+        "min_htlc": "htlc_minimum_msat",
+        "fee_base_msat": "base_fee_millisatoshi",
+        "fee_rate_milli_msat": "fee_per_millionth",
+        "disabled": "active",
+        "max_htlc_msat": "htlc_maximum_msat",
+        "last_update": "last_update",
+    }
+
+    def _add_direction(src, dest, lnd_channel, cln_channel):
+        cln_channel["source"] = lnd_channel[src + "_pub"]
+        cln_channel["destination"] = lnd_channel[dest + "_pub"]
+
+        src_policy = lnd_channel[src + "_policy"]
+        for key in src_policy:
+            if key == "disabled":
+                cln_channel["active"] = src_policy[key]
+            cln_channel[LND_CLN_POLICY_MAP[key]] = int(src_policy[key])
+
+    cln_channel_json = []
+    for lnd_channel in channel_graph_json:
+        # Common fields for both direction
+        cln_channel = {
+            "short_channel_id": lnd2cln_channel_id(int(lnd_channel["channel_id"])),
+            "satoshis": int(lnd_channel["capacity"]),
+            "amount_msat": int(lnd_channel["capacity"]) * 1000,
+            "public": None, # Info not available in lnd describegraph?
```

Comment:
Not entirely sure. I guess it is public if it is in the json? guess we have to check the lnd code repo or ask someone there

Reply:
Thanks, that makes sense 👍
It's set to True in a174641